src/languages/en.js

```diff
@@ -1183,6 +1183,9 @@ export default {
         pleaseEnterTaskAssignee: 'Please select an assignee',
         pleaseEnterTaskDestination: 'Please select a share destination',
     },
+    taskReport: {
+        to: 'To',
+    },
```

Comment:
Let's use the existing 'To' constant for this. I believe it's somewhere in common